assets/sass/patterns/organisms/content-header-simple.scss

```diff
@@ -19,3 +19,72 @@
   @include body-typeg();
   @include nospace();
 }
+
+.clean {
+  @include margin(12, "bottom");
+
+  p {
+    color: $color-text;
+    font-family: $font-secondary;
+
+    a:not(.additional-asset__link--download):not(.asset-viewer-inline__download_all_link):not(.asset-viewer-inline__open_link):not(.reference__title):not(.doi):not(.reference__authors_link):not(.trigger):not(.popup__link):not(.see-more-link) {
+      border-bottom-style: solid;
```

Comment:
I thought that any main content links would be dotted. Underline looks very harsh. I think we need to talk to Chris about this?